internal/metamorph/store/postgresql/migrations/000016_multiple_callbacks.up.sql

```diff
@@ -0,0 +1,17 @@
+-- Step 1: Add the new 'callback' column
+ALTER TABLE metamorph.transactions
+    ADD COLUMN callbacks JSONB;
+
+-- Step 2: Populate the 'callback' column with data from 'callback_url' and 'callback_token'
+UPDATE metamorph.transactions
+SET callbacks = json_build_array(
+    json_build_object(
+        'callback_url', callback_url,
+        'callback_token', callback_token
+    )
+)WHERE callback_url IS NOT NULL OR callback_token IS NOT NULL;
```

Comment:
I think we can't do anything with callback without url so probably we can ignore them

Reply:
I'm also almost 100% sure that we have empty strings instead of null in all records, so it would be better to check the length

Reply:
Done